compiler/src/dmd/dmodule.d

```diff
@@ -1382,7 +1386,7 @@ extern (C++) struct ModuleDeclaration
  * for inclusion in ModuleInfo
  * Params:
  *      mod = the Module
- *	aclasses = array to fill in
+ *      aclasses = array to fill in
```

Comment:
Unrelated change, should be its own commit

Reply:
Another case of tabs being converted to spaces by my checkin procedure. Not worth another PR.